packages/fx-core/src/plugins/solution/fx-solution/debug/util/tasksNext.ts

```diff
@@ -248,7 +248,7 @@ function startBackend(programmingLanguage: string): Record<string, unknown> {
     options: {
       cwd: "${workspaceFolder}/api",
       env: {
-        PATH: "${env:PATH}${command:fx-extension.get-func-path}",
+        PATH: "${command:fx-extension.get-func-path}${env:PATH}",
```

Comment:
How do we handle the existing projects?

Reply:
Another solution 
For node v16:
* If global func v3 installed, stop F5 with error message to ask user upgrade it to v4 manually. 
* If portable func v3 installed, upgrade func to v4 automatically and continue F5.